packages/apollo-vscode/README.md

```diff
@@ -0,0 +1,66 @@
+# Apollo VSCode
+
+An all-in-one tooling experience for developing apps with Apollo
+
+- Get instant feedback and intelligent autocomplete as you write queries
+- Run queries against your GraphQL server without leaving your editor
+- View performance statistics next to your query definitions
+
+![Code completing queries](images/variable-argument-completion.gif)
+![Running a query with variables](images/query-with-vars.gif)
+![Viewing Engine statistics](images/engine-stats.png)
+
+## Features
```

Comment:
Does this extension work with non-Apollo GraphQL servers?

Reply:
Yes! Is there somewhere we can make this more clear?